data/image_train_item.py

```diff
@@ -314,8 +314,10 @@ def __compute_target_width_height(self):
                 image_aspect = width / height
                 target_wh = min(self.aspects, key=lambda aspects:abs(aspects[0]/aspects[1] - image_aspect))
 
-                self.is_undersized = (width * height) < (target_wh[0]*1.02 * target_wh[1]*1.02)
+                self.is_undersized = (width != target_wh[0] and height != target_wh[1]) and (width * height) < (target_wh[0]*1.02 * target_wh[1]*1.02)
+
                 self.target_wh = target_wh
+                self.image_size = image.size
```

Comment:
If the images is actually undersized, we need to populate this field to display the size in the log entry